src/org/jitsi/impl/neomedia/transform/fec/FlexFec03Receiver.java

```diff
@@ -0,0 +1,341 @@
+/*
+ * Copyright @ 2017 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.impl.neomedia.transform.fec;
+
+import org.jitsi.service.neomedia.*;
+import org.jitsi.util.*;
+
+import java.util.*;
+
+/**
+ * Receive and process FlexFec03 packets, recovering missing packets where
+ * possible
+ * @author bbaldino
+ */
+public class FlexFec03Receiver
+    extends AbstractFECReceiver
+{
+    /**
+     * The <tt>Logger</tt> used by the <tt>FlexFec03Receiver</tt> class and
+     * its instances to print debug information.
+     */
+    private static final Logger logger
+        = Logger.getLogger(FlexFec03Receiver.class);
+
+    /**
+     * FEC-related statistics
+     */
+    private Statistics statistics;
```

Comment:
Do we intentionally shadow super.statistics? Looks like super is the one which handles numRxFecPackets

Reply:
nope, think this was probably a copy-paste issue. fixed.